frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,20 @@
+import { type IconName } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/get-valid-class-names.js';
+import { iconNameToPlainSvgMap } from '#libs/maps/icon-name-to-plain-svg.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  name: IconName;
+  className?: string;
+};
+
+const Icon: React.FC<Properties> = ({ name, className = '' }) => {
+  const SelectedIcon = iconNameToPlainSvgMap[name];
+
+  const cssClass = getValidClassNames(styles[className]);
```

Comment:
don't pass className to icon, for properties like color we will pass them as color, for margins, padding we will wrap icon in div outside this component
```suggestion
```

Reply:
@Liza-Veis so how should it be? I tried to add prop 'fill' to SelectedIcon but it does not work. To change fill of the svg i have to change path`s fill, but without css it is impossible

Reply:
if you set fill/color/border/stroke (depends on how color set in svg) in svg to currentColor, then you should be able to set this color using css color property

Reply:
done